app/jobs/submit_iplc_listener_job.rb

```diff
@@ -0,0 +1,91 @@
+# frozen_string_literal: true
+
+##
+# Background job to send requests to IPLC.
+# If Reshare has the item and we succesfully request it, let the user know it is on the way.
+# If Reshare does not have it or we cannot succesfully request it, trigger the normal Symphony request workflow.
+class SubmitIplcListenerJob < ApplicationJob
+  discard_on ActiveRecord::RecordNotFound do |job, _error|
+    Honeybadger.notify(
+      "Attempted to call Reshare for Request with ID #{job.request_id}, but no such Request was found."
+    )
+  end
+
+  def perform(request_id, instance_uuid)
+    request = Request.find(request_id)
+
+    Sidekiq.logger.info("Started SubmitIplcRequestJob for request #{request_id}")
```

Comment:
Why do we need `Sidekiq` here? `logger` is an instance method on `ApplicationJob`, right?

Reply:
I'm guessing it's to support the special log level configuration for Sidekiq?

https://github.com/sul-dlss/sul-requests/blob/1f6e0dd89895cf6cf287d6e0aeb35a151162311f/config/initializers/sidekiq.rb#L3

This is just consistent with the other request jobs :shrug: